cinema/serializers.py

```diff
@@ -114,7 +143,15 @@ class MovieSessionDetailSerializer(MovieSessionSerializer):
 
     class Meta:
         model = MovieSession
-        fields = ("id", "show_time", "movie", "cinema_hall", "taken_places")
+        fields = (
+            "id",
+            "show_time",
+            "movie",
+            "cinema_hall",
+            "taken_places",
+            "image"
+        )
+        read_only_fields = ("id", "image")
```

Comment:
The `MovieSessionDetailSerializer` should not include the `image` field. Please remove this field from the serializer to align with the task requirements, as the `MovieSession` model does not have an `image` attribute.